wandb/sdk/data_types/image.py

```diff
@@ -287,6 +287,8 @@ def _initialize_from_data(
             )
             if hasattr(data, "requires_grad") and data.requires_grad:
                 data = data.detach()  # type: ignore
+            if str(data.dtype) == "torch.uint8":
```

Comment:
let's add a bit of paranoia here :)
```suggestion
            if hasattr(data, "dtype") and str(data.dtype) == "torch.uint8":
```

Reply:
Updated 👍  thank you